packages/react/src/components/List/ListHeader/ListHeader.test.jsx

```diff
@@ -1,37 +1,9 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
-import '@testing-library/jest-dom/extend-expect';
-
 import ListHeader from './ListHeader';
 
 describe('ListHeader', () => {
```

Comment:
Why did we remove these testcases, they're valid ideas to verify I think

Reply:
I just think we can write better tests than 'should render' as thats already tested by using the full List component with a header. I re-added the tests